app/assets/javascripts/qpixel_api.js

```diff
@@ -320,10 +314,8 @@ window.QPixel = {
 
   fetch: async (uri, init) => {
     const defaultHeaders = {
-      'X-CSRF-Token': QPixel.csrfToken(),
       // X-Requested-With is necessary for request.xhr? to work
       'X-Requested-With': 'XMLHttpRequest',
-      'Content-Type': 'application/json',
```

Comment:
This is a partially related fix that ensures `QPixel#fetch` can be used with non-JSON (`Content-Type` request header) requests to avoid issues with forgetting to set `X-Requested-With` or forwarding credentials. Should not break anything